src/condor_tests/test_negotiator_slot_const.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env pytest
+
+
+import logging
+
+from ornithology import *
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+
+# Plan for this test is to stand up a condor with 20
+# static slots, and configure the NEGOTIATOR_SLOT_CONSTRAINT
+# to only look at slot 7 (randomly chosen).  We then
+# submit a sleep 0 job, wait for it to complete, leave
+# it in the queue, and check to see that it ran on slot7.
+
+@standup
+def condor(test_dir):
+    with Condor(test_dir / "condor", 
+            config={
+                "NUM_CPUS": "20",
+                # What? You got a better hack to handle this?
```

Comment:
Yes; the named argument `raw_config`.

Reply:
(The `raw_config` named argument is used in test_condor_now, test_condor_now_internals, test_drain_policies, test_htcondor_971, and test_htcondor_993, in terms of finding an example.  It even directly sets a metaknob in test_condor_now.)